packages/gerber-parser/test/gerber-parser_drill_test.js

```diff
@@ -605,4 +605,18 @@ describe('gerber parser with drill files', function() {
       })
     })
   })
+
+  describe('parsing altium plating comments', function() {
+    it('should set hole plating', function(done) {
+        var expected = [
+          {type:'set', line:1, prop:'holePlating', value:'pth' },
```

Comment:
Looks like these lines are failing lint. An `npm run format` should fix that up

Reply:
My bad. Done.